cli/_cli.php

```diff
@@ -75,6 +75,48 @@ function performRequirementCheck(string $databaseType): void {
 	}
 }
 
+/**
+ * Parses perameters used with FreshRSS' CLI commands.
+ * @param array{'valid':array<string,string>,'deprecated':array<string,string>} $parameters An array of 'valid': An
+ * array of perameters as keys and their respective getopt() notations as values. 'deprecated' An array with
```

Comment:
```suggestion
 * array of parameters as keys and their respective getopt() notations as values. 'deprecated' An array with
```

There are multiple instances of `parameters`. Please replace them by `parameters`.

Reply:
That's fixed now.

Reply:
Can't believe I forgot how to spell parameters...